examples/LLM/llama2/chat_app/torchserve_server_app.py

```diff
@@ -0,0 +1,200 @@
+import json
+import os
+
+import requests
+import streamlit as st
+
+MODEL_NAME = "llamacpp"
+# App title
+st.set_page_config(page_title="🦙💬 Llama 2 TorchServe Serve")
+
+
+def start_server():
+    os.system("torchserve --start --model-store model_store --ncs")
+    st.session_state.started = True
+    st.session_state.stopped = False
+    st.session_state.registered = False
+
+
+def stop_server():
+    os.system("torchserve --stop")
```

Comment:
add some simple log statements for each of hte main functions

Reply:
The app has statements like these `server_state_container.success("Started TorchServe", icon="✅")` These show up on the app

Reply:
This exists in the app